File: src/server/datalayer/branch.cpp
#include "branch.h"

Branch::Branch() {
    name = "";
}

Branch::Branch(std::string _name) {
    name = _name;
}

Branch::~Branch() { }

int Branch::getId() {
    return id;
}

void Branch::setName(std::string _name) {
    name = _name;
}

std::string Branch::getName() {
    return name;
}

File: src/web/Models/Branch.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace VehicleTracker.Models
{
    public class Branch
    {
        private String id;
        private String name;

        [StringLength(10, ErrorMessage = "Tối đa 10 ký tự")]
        [Required(ErrorMessage = "Không để trống trường này")]
        public string Id { get => id; set => id = value; }

        [StringLength(100, ErrorMessage = "Tối đa 100 ký tự")]
        [Required(ErrorMessage = "Không để trống trường này")]
        public string Name { get => name; set => name = value; }
    }
}
